### PR TITLE
Tickets Emails - Dynamizing Attendees & Improving Placeholders

### PR DESCRIPTION
**Pull request by codingmusician**
### 🗒️ Description

This is a WIP. We need to dynamize the attendee data for the Completed Order, Failed Order and Purchase Receipt emails. I also realized that placeholders weren't being set in the preview data, so I'm also working on enhancing that experience so the template more reflect real-life data.

### 🎥 Artifacts 

https://www.loom.com/share/c5289387ae74410fb8792eb5ac573adf

### ✔️ Checklist
- [ ] I've included a changelog entry in the readme.txt file. 
- [x] My code is tested. 
- [x] My code has [proper inline documentation](https://the-events-calendar.github.io/products-engineering/docs/code-standards/).

### CODE REVIEW

File: src/Tickets/Emails/Email/Ticket.php [hunk 1]
```diff
@@ -305,6 +306,12 @@ public function get_settings_fields(): array {
 	public function get_default_preview_context( $args = [] ): array {
 		$defaults = tribe( Email_Template::class )->get_preview_context( $args );
 
+		$args['order'] = Preview_Data::get_order();
+		$args['tickets'] = Preview_Data::get_tickets();
+		$args['heading'] = count($args['tickets']) > 1 ?
+			$this->get_heading_plural() :
+			$this->get_heading();
```
Comment (bordoni) [inline src/Tickets/Emails/Email/Ticket.php:313]: We should avoid ternaries like this.

**Review by bordoni — CHANGES_REQUESTED** [review]

**Comment by codingmusician** [comment]
To Do: Need to update all references to `__set()` and `__get()` once #2615 is merged.

**Comment by bordoni** [comment]
You can update the `__set()` and `__get()` refs, it's merged.

**Comment by codingmusician** [comment]
Still a WIP: There's currently a discrepancy between the `tickets` data that get passed into `Ticket` email versus the other emails.

**Comment by redscar** [comment]
Looks like one of the tests are failing

**Review by bordoni — DISMISSED** [review]

**Pull request merged**